build.gradle

```diff
@@ -50,6 +50,10 @@ allprojects {
         tasks.withType(JavaCompile) {
             options.compilerArgs += ['-Werror', '-Xlint:deprecation']
             options.errorprone {
+                // Temporarily disable errorprone, because it cannot pass due to the issue that is being fixed
+                // in #3101. Remove this line after a new version has been released.
+                getDisableAllChecks().set(true)
```

Comment:
Is it possible to make this a bit more targeted? Or are all checks broken?

https://github.com/palantir/gradle-baseline/commit/f9539d53d1a1bfd945e9fc9f1db8b499e73cdbf1#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R55-R56